sdk/oracle/Azure.ResourceManager.OracleDatabase/Azure.ResourceManager.OracleDatabase.sln

```diff
@@ -0,0 +1,65 @@
+﻿
```

Comment:
@ArthurMa1978 all the other languages have this under `sdk/oracledatabase` is there any reason .NET cannot follow the same?